lib/cylc/daemonize.py

```diff
@@ -52,13 +68,53 @@ def daemonize(server):
     http://code.activestate.com/recipes/66012-fork-a-daemon-process-on-unix/
 
     """
-
-    sout = SuiteOutput(server.suite)
-
+    logd = SuiteLog.get_dir_for_suite(server.suite)
+    log_fname = os.path.join(logd, SuiteLog.LOG)
+    try:
+        old_log_mtime = os.stat(log_fname).st_mtime
+    except OSError:
+        old_log_mtime = None
+    # fork 1
     try:
         pid = os.fork()
         if pid > 0:
-            # exit first parent
+            # Poll for suite log to be populated
+            suite_pid = None
+            suite_port = None
+            while suite_pid is None or suite_port is None:
```

Comment:
Should this loop time out after a while in case something stops the log file being written?